plugins/modulbank/main.js

```diff
@@ -0,0 +1,397 @@
+// chukaev@live.ru +79991554646
+
+var g_headers = {},
+	g_authurl =  "https://oauth.modulbank.ru/api/oauth2/",
+	g_baseurl =  "https://api.modulbank.ru/v1/",
+	g_sessionid,
+	g_preferences;
+
+/**
+ * Основной метод
+ */
+function main(){
+	
+	g_preferences = ZenMoney.getPreferences();
+	
+	login();
+
+	processAccounts();
+	processTransactions();
+	ZenMoney.setResult({success: true});
+}
+
+/**
+ * Авторизация
+ * @returns {*}
+ */
+function login() {
+
+	if (!g_preferences.login) 
+
+		throw new ZenMoney.Error("Введите логин в интернет-банк!", null, true);
+	
+	if (!g_preferences.password) 
+
+		throw new ZenMoney.Error("Введите пароль в интернет-банк!", null, true);
+
+	if (g_sessionid){
+
+		ZenMoney.trace("Cессия уже установлена. Используем её.");
+		
+		return;
+	}
+
+	var code = ZenMoney.requestPost(g_authurl, {
+		"CellPhone":g_preferences.login,
+		"Password":g_preferences.password,
+		"ClientId":g_preferences.clientId,
```

Comment:
You will need to register by tech support manager in modul bank to get 
clientId and 
clientSecret
I has registered app with same name as zenmoney and you can use that if you want
for details chukaev@live.ru